scss/typography/_helpers.scss

```diff
@@ -38,6 +38,32 @@ $subheader-margin-bottom: 0.5rem !default;
 /// @type Number
 $stat-font-size: 2.5rem !default;
 
+/// If `true`, makes dropcaps uppercase.
+/// @type Boolean
+$dropcaps-uppercase: true !default;
+
+/// Dropcaps component
+@mixin dropcaps($uppercase: $dropcaps-uppercase) {
+  // Default margin (Arial/Helvetica) to let it properly aligned from the top on the 
+  // first line beside it and have a tiny space between them through the right
+  $-margin: 0.192em 0.0962em 0 0; 
```

Comment:
Hmm... Just seen it is not bidi-compliant (read in RTL and LTR).

We can use this instead:
```sass
$-margin-top: 0.192em;
$-margin-right: 0.0962em;

...

margin-top: $-margin-top;
margin-#{$global-right}: $-margin-right;
```